cli/app.go

```diff
@@ -75,6 +75,7 @@ const (
 	moduleFlagForce           = "force"
 	moduleFlagBinary          = "binary"
 	moduleFlagLocal           = "local"
+	moduleFlagNoProgress      = "no-progress"
```

Comment:
(minor) can we collapse this and `cpFlagNoProgress` into a single flag (`generalFlagNoProgress` or something like that)? Would prefer to not have two identical flags.

Reply:
Sure thing! I did not see the general flags above earlier. Makes total sense.